game/core/src/main/java/com/gg/game/model/levels/Wall.java

```diff
@@ -8,7 +9,7 @@
  * Represents the outer boundary of a room.
  */
 class Wall implements Collidable {
-    private static final int DEFAULT_WALL_SIZE = 50;
+    private static final int DEFAULT_WALL_SIZE = Constant.WALL_SIZE;
```

Comment:
```suggestion
    private static final int WALL_SIZE = Constant.WALL_SIZE;
```

No need for longer field name. Refactor uses as well.